src/DevTools/setupTestWrapper.tsx

```diff
@@ -138,11 +138,11 @@ export const setupTestWrapper = <T extends OperationType>({
 }: SetupTestWrapper<T>) => {
   const getWrapper = (
     mockResolvers: MockResolvers = {},
-    componentProps: {} = {}
+    componentProps: {} = {},
+    mockedEnv?: ReturnType<typeof createMockEnvironment>
   ) => {
-    const env = createMockEnvironment()
+    const env = mockedEnv ?? createMockEnvironment()
```

Comment:
the only change in our relay tools I've had to make, we'll revisit that in the future for sure as soon as we introduce hooks and have mixed Fragment/Hooks/etc in the same file, the test probably will need to control the env outside for handling data/dealing with mutations, this is open to suggestion but we should enforce the same pattern to use in all our React-Relay fronts, Force, Eigen, etc.